lighthouse-core/gather/driver.js

```diff
@@ -1094,11 +1105,13 @@ class Driver {
       await this._waitForFrameNavigated();
     } else if (waitForLoad) {
       const passConfig = /** @type {Partial<LH.Config.Pass>} */ (passContext.passConfig || {});
-      let {pauseAfterLoadMs, networkQuietThresholdMs, cpuQuietThresholdMs} = passConfig;
+
+      /* eslint-disable max-len */
+      let {pauseAfterFcpMs, pauseAfterLoadMs, networkQuietThresholdMs, cpuQuietThresholdMs} = passConfig;
       let maxWaitMs = passContext.settings && passContext.settings.maxWaitForLoad;
       let maxFCPMs = passContext.settings && passContext.settings.maxWaitForFcp;
 
-      /* eslint-disable max-len */
+      if (typeof pauseAfterFcpMs !== 'number') pauseAfterFcpMs = DEFAULT_PAUSE_AFTER_FCP;
```

Comment:
we should really clean this up as well, after all the effort we go through to get our settings and constants in order before loading a page :)

It looks like these will not be set only if a `passContext` wasn't passed in, and we don't pass in a `passContext` only when we're loading `blankPage`, which ends up mostly doing `_waitForNothing`, so it doesn't need defaults for these anyways :)